README.md

```diff
@@ -332,29 +297,33 @@ Make your script executable with `sudo chmod a+x /etc/local.d/grub-btrfs-update.
 * The extension `.stop` at the end of the filename indicates to the locald-daemon that this script should be run at shutdown. 
  If you want to run the menu update on system startup instead, rename the file to `grub-btrfs-update.start`
 * Works for Snapper and Timeshift
+
+[Here is a similar solution for systemd](https://github.com/Antynea/grub-btrfs/issues/138#issuecomment-766918328)
```

Comment:
you wanna make your guide into a first class citizen and put it in readme directly?

Reply:
I was only using my regenerate GRUB on shutdown script because grub-btrfsd wasn't working for me (under Debian) at the time. Personally I'd advise people not to use that method so I don't feel right adding a link to my guide.

Reply:
yes, I was thinking the same thing when I dropped the instructions for this. Why have it when grub-btrfsd works? 
Also this is basically a instruction for systemd service files and not our business. 

Shall we drop the instructions for openrc's locald is well then? Seems weird to have it in there and the same thing for systemd not.

Reply:
Makes sense to me.